css/values.css

```diff
@@ -38,14 +28,6 @@ Hero
 	display: flex;
 	justify-content: flex-end;
 }
-/* values heading sizing */
-.hero-text-container {
```

Comment:
Is this waiting for my Events change?

<img width="1666" alt="Screen Shot 2024-01-12 at 7 37 05 AM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/0548fc01-b87c-4ecb-8da9-982cfc0720c9">

Reply:
yes this was removed and waiting for events page PR.